src/lib/xpubs.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ @see https://github.com/blockkeeper/blockkeeper-frontend-web/issues/38
+ */
+import { UtxoAccountType } from '@shapeshiftoss/types'
+import { decode, encode } from 'bs58check'
```

Comment:
nitpick: the names "encode" and "decode" aren't very specific, especially since there's `bs58` library which has functions with the same names but doesn't do the checksumming. to avoid `import { decode as bs58decode, encode as bs58encode } from 'bs58check'` i suggest `* as bs58check`.

```suggestion
import * as bs58check from 'bs58check'
```